tools/recipes/rt-app.recipe

```diff
@@ -1,5 +1,7 @@
 #! /bin/bash
 
+ALPINE_BUILD_DEPENDENCIES=(autoconf automake bash bison gcc gettext git flex libtool linux-headers make musl-dev numactl-dev)
```

Comment:
Do we actually need `numactl-dev` and `linux-headers` to compile rt-app ?
`numactl` is compiled inside this recipe already, and rt-app ships with some syscall headers, which is the only use I can see for kernel headers in rt-app.

Things like bison and flex also seem weird, we don't build anything that has a parser in it except the JSON one, which is a hand-written recursive parser (json-c)

Reply:
There is a dependency over the linux-headers in rt-app/libdl/dl_syscalls.h
But you were right, we don t need all of this.